Altis_Life.Altis/config/Config_Vehicles.hpp

```diff
@@ -578,60 +623,60 @@ will modify the virtual space and the price of the vehicle, but other informatio
         textures[] = {
             { "Rebel Digital", "reb", {
                 "\a3\air_f\Heli_Light_01\Data\Skins\heli_light_01_ext_digital_co.paa"
-            } }
+            }, "" }
         };
     };
 
     class B_Heli_Light_01_F {
         vItemSpace = 90;
-        conditions = "license_civ_pilot || {license_cop_cAir} || {license_med_mAir}";
+        conditions = "license_civ_pilot || {license_cop_cg} || {license_civ_mAir}";
```

Comment:
Police need a coast guard license in order to buy the Hummingbird. _Appears to be legitimate._ :kappa: